apps/laboratory/src/utils/SiweUtils.ts

```diff
@@ -1,20 +1,19 @@
-import { SiweMessage } from 'siwe'
 import { getCsrfToken, signIn, signOut, getSession } from 'next-auth/react'
 import type { SIWEVerifyMessageArgs, SIWECreateMessageArgs, SIWESession } from '@web3modal/siwe'
 import { createSIWEConfig } from '@web3modal/siwe'
+import { formatMessage } from '@walletconnect/utils'
```

Comment:
mmm I see we are importing a lot of functions and utilies from @walletconnect/utils, what if we export them from `@web3modal/siwe`? so devs would only need to install siwe instead of siwe + utils?

Reply:
@ganchoradkov I forgot to move this to the SIWE package. We could import it from there too